Makefile

```diff
@@ -663,6 +663,7 @@ isti%.yaml: $(HELM) $(HOME)/.helm
 		--set global.proxy.enableCoreDump=${ENABLE_COREDUMP} \
 		--set istio_cni.enabled=${ENABLE_ISTIO_CNI} \
 		${EXTRA_HELM_SETTINGS} \
+		--values install/kubernetes/helm/istio/values-e2e.yaml \
```

Comment:
can this be done elsewhere?
when someone runs `make istio-auth.yaml` they do not expect to get e2e overrides. 
We need to fix it in a different way.

Reply:
/hold

Reply:
Where else would this be done? These files are all only generated for e2e testing. The targets below, generate_yaml and generate_e2e_yaml are can be completely replaced with one target that calls the updateVersion if we do this (follow up pr). Right now we generate half of the files with the values-e2e applied and half without for no apparent reason.

If we really want to be able to generate these without the values-e2e settings applied, we could make two targets but I don't see a need - we wouldn't use the ones without values-e2e applied anywhere.

Reply:
Ok actually now that I look at it longer, we do have some issues.

This one rule makes `istio-demo`, all the e2e values, examples, etc